src/pybamm/plotting/quick_plot.py

```diff
@@ -479,24 +479,24 @@ def reset_axis(self):
             ):  # pragma: no cover
                 raise ValueError(f"Axis limits cannot be NaN for variables '{key}'")
 
-    def plot(self, t, dynamic=False):
+    def plot(self, t: float | list[float], dynamic: bool = False):
         """Produces a quick plot with the internal states at time t.
 
         Parameters
         ----------
-        t : float
-            Dimensional time (in 'time_units') at which to plot.
+        t : float or list of float
+        Dimensional time (in 'time_units') at which to plot. Can be a single time or a list of times.
```

Comment:
This needs to be indented to render properly
```suggestion
            Dimensional time (in 'time_units') at which to plot. Can be a single time or a list of times.
```

Reply:
You can see this if you check the "readthedocs" output in the checks section. It is good to check the docs whenever you update them